source/gloperate/source/base/Canvas.cpp

```diff
@@ -151,42 +151,42 @@ void Canvas::onBackgroundColor(float red, float green, float blue)
 
 void Canvas::onKeyPress(int key, int modifier)
 {
-    cppassist::debug(1) << "onKeyPressed(" << key << ")";
+    cppassist::debug(2) << "onKeyPressed(" << key << ")";
```

Comment:
The modifier is important, too.
Unfortunately, this is a bitmask and the most information is obtained when converting the bitmask to an explicit enum values representation.